### DISCUSSION TITLE
''yo theia-extension'' failed

### DISCUSSION THREAD

**Discussion by Adem-Kri** [eclipse-theia/theia#11813] [General] [answered]
I am following the Theia documentation, and everything worked as intended until I reached the `/docs/authoring_extensions/` section. So it crashes when I type `yo theia-extension`.
As you can see, instructions work fine until they reach the `browser-app`, and everything begins here:
![image](https://user-images.githubusercontent.com/116633370/198556903-a9e3407d-5d89-42f4-ac26-fb0791bf82fa.png)

it shows a bunch of browser-app failures
![image](https://user-images.githubusercontent.com/116633370/198557037-83192de5-aa5c-45a1-8b73-fdc7e4fe8fd5.png)
I doubt in this `RequestError: getaddrinfo ENOTFOUND github.com`
![image](https://user-images.githubusercontent.com/116633370/198557093-b6a7d6f6-1f29-441d-98d3-12b995323832.png)

**Comment by msujew** [comment]
Hey @Adem-Kri,

are you behind a corporate proxy? Some of the dependencies of Theia require to download additional files from github in order to build correctly. A proxy might block these requests from happening.

**Reply by Adem-Kri** [reply]
Hello @msujew,
yes I am behind my company's proxy, I have used this: `git config --global http.proxy http://login:password@company's_domain:port(8080)`
could you give me the appropriate command to execute and bypass the proxy .
thank you.

**Reply by msujew** [reply]
The node.js dependencies in question don't use the git proxy. I believe they look for the `http_proxy` or `https_proxy` environment variables, but I'm not totally sure whether they correctly support proxies at all.

**Comment by Adem-Kri** [comment, marked as answer]
Hello @msujew, I found the solution to my issue. I made a mistake by installing yarn multiple times, first with scoop then with npm. So, even after setting the Proxy correctly, when executing the command, it will be confused between the two versions. That's what I have understood.
I want to thank you for your time and if I don't bother I have another problem which am gonna write in another discussion.
